types/promise.any/OTHER_FILES.txt

```diff
@@ -0,0 +1,2 @@
+auto.d.ts
+requirePromise.d.ts
```

Comment:
```suggestion
requirePromise.d.ts
```
I'd say we don't need this, it's just runtime feature watchdog:
https://github.com/es-shims/Promise.any/blob/44f6dbee034a4f97ca10b879c31a07d7eb17c1da/requirePromise.js

Reply:
The `requirePromise` watchdog is used internally by the polyfill function (exported by shim.js) to make sure that there is a `Promise` class to polyfill onto. Since this file is exported, and there is a chance that users might import it (for whatever reason), and [@types/promise.allsettled](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/promise.allsettled) does the same, I opted to include requirePromise.d.ts in OTHER_FILES.txt.

`promise.allsettled` does the same in their [OTHER_FILES.txt](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/promise.allsettled/OTHER_FILES.txt). I'm not sure that we need to use a different pattern here.